app/components/markdown/index.ts

```diff
@@ -4,19 +4,16 @@
 import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
 import withObservables from '@nozbe/with-observables';
 import React from 'react';
-import {of as of$} from 'rxjs';
-import {switchMap} from 'rxjs/operators';
 
-import {observeConfig} from '@queries/servers/system';
+import {observeConfigBooleanValue} from '@queries/servers/system';
 
 import Markdown from './markdown';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
 
 const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
-    const config = observeConfig(database);
-    const enableLatex = config.pipe(switchMap((c) => of$(c?.EnableLatex === 'true')));
-    const enableInlineLatex = config.pipe(switchMap((c) => of$(c?.EnableInlineLatex === 'true')));
+    const enableLatex = observeConfigBooleanValue(database, 'EnableLatex');
+    const enableInlineLatex = observeConfigBooleanValue(database, 'EnableInlineLatex');
```

Comment:
Not sure if there are real performance implications here (probably not) but here we have two different observers for `config` while previously we had only one observer for config, connected to two different pipes. Nothing to do here, but we should keep in mind in case it becomes a performance issue.